src/post_section/post_section.jsx

```diff
@@ -13,13 +14,13 @@ const links = [
         text: "Share on Twitter"
     },
     {
-        name: "phone abyss",
+        name: "phone",
         src: "/photos/phone.png",
         text: ""
     }
 ];
 function LinksListBottom({ link }) {
-    const className = link.name === "phone" ? link.name : link.name + " _bottom_table"
+    const className = link.name === "phone" ? link.name : link.name + "_bottom_table"
```

Comment:
I don't really like that every link has it's own class based on the `name` attr. don't all the links look the same? can you use the same class for the all

for the `_bottom_table` part, you can add a new boolean attr to the objects in the array and if that attr is true, then append another class